### PR TITLE
Parse SRID in column definition

### PR DESCRIPTION
**Pull request by MoonE**
Fixes https://github.com/phpmyadmin/phpmyadmin/issues/18513

The parsing does not consider the db version and MariaDB does not support this.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
Awesome

File: tests/data/parser/parserCreateTableSRID.in [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS `public_areas` (
```
Comment (williamdes) [inline tests/data/parser/parserCreateTableSRID.in:1]: Are we not missing a line in the test file for the test to pick up this file?

Comment (MoonE) [reply tests/data/parser/parserCreateTableSRID.in:1]: Is picked up automatically:
https://github.com/phpmyadmin/sql-parser/blob/98515793e67df12da89ebef2be4d014f09361e80/src/Tools/TestGenerator.php#L234

Comment (MoonE) [reply tests/data/parser/parserCreateTableSRID.in]: You are right, only the generation of the out file is picked up automatically. I added the missing line.

**Review by williamdes — APPROVED** [review]
💯

**Review by MauricioFauth — APPROVED** [review]
Thank you!

**Pull request merged**